shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -12,10 +12,14 @@
 from shimmingtoolbox.utils import create_output_dir
 
 
-def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', path_config_dcm2bids=__dir_config_dcm2bids__, remove_tmp=False):
+def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', path_config_dcm2bids=__dir_config_dcm2bids__,
+                   remove_tmp=False):
     """ Converts dicom files into nifti files by calling dcm2bids
 
     Args:
+        path_config_dcm2bids:
+        subject_id:
+        remove_tmp:
```

Comment:
Those are empty

Reply:
Filled in [19307ad](https://github.com/shimming-toolbox/shimming-toolbox/pull/338/commits/19307add8b95c6c8e469e84b90fad94aafc11958)